### ISSUE TITLE
FEAT: horizontal scroll on tabs using mouse

### ISSUE THREAD

**Issue by mcanepa** [beekeeper-studio/beekeeper-studio#1403]
If there are too many tabs, I would expect two things

1) Each of them to have a minimum with so at least part of it's name gets shown. See #1401

2) To see a certain number of tabs and scroll left or right with mouse wheel in order to access the rest

**Comment by loomchild** [comment]
Yeah, it would be great. Otherwise, the workspace eventually becomes unusable and the only solution is to close all tabs and start from scratch:

![Image](https://github.com/user-attachments/assets/434378ea-9ecd-4547-8048-8702a691d436)

**Comment by mcanepa** [comment]
Please see [https://github.com/beekeeper-studio/beekeeper-studio/pull/3706#issuecomment-5269483713](https://github.com/beekeeper-studio/beekeeper-studio/pull/3706#issuecomment-5269483713)

@rathboma @not-night-but @azmy60
